src/global-messages/GlobalMessage.tsx

```diff
@@ -39,10 +41,14 @@ const GlobalMessage = ({
 
   if (!globalMessageContext) return null;
 
-  const globalMessages = findGlobalMessages(
-    globalMessageContext,
-    ruleVariables,
-  );
+  const globalRuleVariables = {
```

Comment:
Makes sense to include the typing here, so it's clear that it should be on the RuleVariable format in the future.

```suggestion
  const globalRuleVariables: RuleVariables = {
```